frontend/src/stories/Search.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import { TextField } from "../components";
+import clsx from "clsx";
+
+export const Normal = () => (
```

Comment:
This story seems to just be a copy/paste of the Search bar component.

Reply:
Honestly don't know how to make stories. Any documentation you can recommend?

Reply:
Just have a look at the other ones. Copy and paste the entire file, delete all the extra stories you don't need, and swap out the components.